app/scripts/modules/core/src/search/infrastructure/infrastructure.controller.js

```diff
@@ -17,6 +19,7 @@ import { ClusterState } from 'core/state';
 
 import { SearchService } from '../search.service';
 import { ConfigureProjectModal } from 'core/projects';
+import {InsightMenu as SearchInsightMenu} from 'core/insight/InsightMenu';
```

Comment:
This looks like it doesn't match the formatting style. We have a post-commit hook that runs `prettier`, did it somehow fail to run?

From the command line, you can run ```npx prettier --write app/scripts/modules/core/src/search/infrastructure/infrastructure.controller.js``` and see if it updates the formatting.  If so, it'd be great to run prettier on all the files in this PR.

Reply:
Yep, just ran and pushed up a squashed commit w/ the prettified files. Thanks!